aten/src/ATen/native/cuda/jit_utils.cpp

```diff
@@ -186,65 +190,81 @@ struct alignas(2) BFloat16 {
 }
 )ESCAPE";
 
-//copy-pasted from util/complex.h
-const std::string jiterator_complex_support_literal = R"ESCAPE(
-//a very limited complex class, the only thing it currently allows is implicit conversion
-//to complex, and complex -> real that is unused
-namespace c10 {
-  template<typename T>
-  struct alignas(sizeof(T) * 2) complex {
-    using value_type = T;
-
-    T real_ = T(0);
-    T imag_ = T(0);
-    constexpr complex() = default;
-    inline __host__ __device__ constexpr complex(const T& re, const T& im = T())
-      : real_(re), imag_(im) {}
-
-    //FIXME I didn't find how complex -> real conversion is done in eager
-    //we are not going to use it, but it's needed for compilation
-    inline __host__ __device__ operator T() const{
-      return real_;
+const std::string dynamic_cast_support_literal = R"ESCAPE(
```

Comment:
Comment here with where this code is copy-pasted from

Reply:
Added